demos/python_demos/action_recognition/action_recognition_demo/models.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, model_xml, model_bin, ie_core, target_device, num_requests, b
 
         # Read IR
         print("Reading IR...")
-        self.net = IENetwork(model_xml, model_bin)
+        self.net = IECore().read_network(model_xml, model_bin)
```

Comment:
Please, reuse `ie_core`.
And you won't need `from openvino.inference_engine import IECore`.